<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>signup</class>
 <widget class="QWidget" name="signup">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>378</width>
    <height>373</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>60</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>username</string>
   </property>
  </widget>
  <widget class="QPlainTextEdit" name="UsernameInput">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>60</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>enter name</string>
   </property>
  </widget>
  <widget class="QPlainTextEdit" name="PasswordInput">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>120</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>enter password</string>
   </property>
  </widget>
  <widget class="QPlainTextEdit" name="PhoneInput">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>180</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="plainText">
    <string/>
   </property>
   <property name="placeholderText">
    <string>enter phone</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>120</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>password</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>180</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>phone</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>240</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>discount code</string>
   </property>
  </widget>
  <widget class="QPlainTextEdit" name="CodeInput">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>240</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>enter code</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>60</y>
     <width>16</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color : 'red'</string>
   </property>
   <property name="text">
    <string>*</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>120</y>
     <width>16</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color : 'red'</string>
   </property>
   <property name="text">
    <string>*</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>180</y>
     <width>16</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color : 'red'</string>
   </property>
   <property name="text">
    <string>*</string>
   </property>
  </widget>
  <widget class="QCommandLinkButton" name="BackBtn">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>71</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>back</string>
   </property>
  </widget>
  <widget class="QPushButton" name="RegisterBtn">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>300</y>
     <width>99</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>Register</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>381</width>
     <height>381</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <zorder>label_8</zorder>
  <zorder>label</zorder>
  <zorder>UsernameInput</zorder>
  <zorder>PasswordInput</zorder>
  <zorder>PhoneInput</zorder>
  <zorder>label_2</zorder>
  <zorder>label_3</zorder>
  <zorder>label_4</zorder>
  <zorder>CodeInput</zorder>
  <zorder>label_5</zorder>
  <zorder>label_6</zorder>
  <zorder>label_7</zorder>
  <zorder>BackBtn</zorder>
  <zorder>RegisterBtn</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
